api/host.go

```diff
@@ -105,9 +105,13 @@ func ModifyHostStatus(ctx context.Context, env evergreen.Environment, queue ambo
 			return "", http.StatusInternalServerError, errors.New(HostTerminationQueueingError)
 		}
 
+		reason := notes
```

Comment:
Should this actually be handled also outside of HostTerminated so it gets hit at 117?